bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
We should at some point make an "official" decision on if we support packages with spaces in or not -- I don't mind banning them, but I get annoyed by software where spaces in filenames kind-of works in some places, then crashes horribly in others. Once we've made a decision for GAP as a whole, we can fix the script one way or the other -- but that pull request can come later.